src/Blocks/BlockPackedIce.h

```diff
@@ -0,0 +1,39 @@
+
+#pragma once
+
+#include "BlockHandler.h"
+
+
+
+
+
+class cBlockPackedIceHandler :
+		public cBlockHandler
+{
+	using Super = cBlockHandler;
+
+public:
+
+	using Super::Super;
+
+private:
+
+	virtual cItems ConvertToPickups(NIBBLETYPE a_BlockMeta, cBlockEntity * a_BlockEntity, const cEntity * a_Digger, const cItem * a_Tool) const override
+	{
+		// Only drop self when using silk-touch:
+		if (ToolHasSilkTouch(a_Tool))
+		{
+			return cItem(m_BlockType);
+		}
+		else
```

Comment:
Since there's a return already, this else can be removed, I suppose.

Reply:
But the else is run if the tool was't with Silk touch

Reply:
No i got it